source/material_model/rheology/visco_plastic.cc

```diff
@@ -98,7 +101,8 @@ namespace aspect
                                        const unsigned int i,
                                        const std::vector<double> &volume_fractions,
                                        const std::vector<double> &phase_function_values,
-                                       const std::vector<unsigned int> &n_phases_per_composition) const
+                                       const std::vector<unsigned int> &n_phases_per_composition,
+                                       const std::vector<unsigned int> &fluid_pressures) const
```

Comment:
To start with and for testing purposes, should we just call this new term `additional_pressures`?